dockers/docker-fpm-frr/docker_init.sh

```diff
@@ -86,13 +86,15 @@ elif [ "$CONFIG_TYPE" == "split-unified" ]; then
     rm -f /etc/frr/bgpd.conf /etc/frr/zebra.conf /etc/frr/staticd.conf
     write_default_zebra_config /etc/frr/frr.conf
 elif [ "$CONFIG_TYPE" == "unified" ]; then
+    echo "" >  /etc/frr/frr.conf
     CFGGEN_PARAMS=" \
         -d \
         -y /etc/sonic/constants.yml \
         -t /usr/share/sonic/templates/gen_frr.conf.j2,/etc/frr/frr.conf \
     "
     sonic-cfggen $CFGGEN_PARAMS
     echo "service integrated-vtysh-config" > /etc/frr/vtysh.conf
+    echo "fpm address 127.0.0.1" >>  /etc/frr/frr.conf
```

Comment:
Can this added in the frr.conf.j2 itself to keep all frr.conf contents in one file?

Reply:
ok. I will add it to frr.conf.j2